src/screens/FeedScreen/Header.js

```diff
@@ -214,16 +210,18 @@ const _renderAnonimity = ({
                   }}>
                   <AnonymousUsername version={version} anonUserInfo={anonUserInfo} isFeed={true} />
 
-                  {!isSelf && (
-                    <React.Fragment>
-                      <View style={styles.point} />
-                      <TouchableOpacity onPress={() => onPressFollUnFoll(isFollow)}>
-                        <Text style={isFollow ? styles.textFollowing : styles.textFollow}>
-                          {isFollow ? 'Following' : 'Follow'}
-                        </Text>
-                      </TouchableOpacity>
-                    </React.Fragment>
-                  )}
+                  {disabledFollow
+                    ? null
+                    : !isSelf && (
+                        <React.Fragment>
+                          <View style={styles.point} />
+                          <TouchableOpacity onPress={() => onPressFollUnFoll(isFollow)}>
+                            <Text style={isFollow ? styles.textFollowing : styles.textFollow}>
+                              {isFollow ? 'Following' : 'Follow'}
+                            </Text>
+                          </TouchableOpacity>
+                        </React.Fragment>
+                      )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional rendering of the follow button based on `disabledFollow` and `isSelf` is a good approach to control the visibility of the button. However, it's important to ensure that the `disabledFollow` prop is always passed to this component from its parent, otherwise it might cause unexpected behavior.



<!-- This is an auto-generated comment by OSS CodeRabbit -->